ios/Classes/MsalAuthPlugin.swift

```diff
@@ -235,7 +236,7 @@ extension MsalAuthPlugin {
             result(true)
         })
     }
-    
+
```

Comment:
Please rollback this unnecessary change.

Reply:
Check the new code now?